### PR TITLE
Release version 2.1.1 of conjury

### PR DESCRIPTION
**Pull request by jhwoodyatt**

### CODE REVIEW

**Comment by jhwoodyatt** [comment]
And alas, it looks like `omake.0.10.5` is _also_ incompatible with `ocaml.5.0`. I filed an issue with `omake` for it.

File: packages/conjury/conjury.2.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+authors: "james woodyatt <jhw@conjury.org>"
+license: "BSD-2-Clause"
+tags: "org:conjury.org"
+homepage: "https://bitbucket.org/jhw/conjury/"
+bug-reports: "https://bitbucket.org/jhw/conjury/issues"
+depends: [
+  "base-unix"
+  "ocaml" { >= "4.08"}
+  "ocamlfind" { >= "1.7.3"}
+  "omake" { = "0.10.3"}
```
Comment (mseri) [inline packages/conjury/conjury.2.1.1/opam:13]:
```suggestion
  "omake" { = "0.10.5"}
```

Comment (jhwoodyatt) [reply packages/conjury/conjury.2.1.1/opam:13]: Ah. I think I see my mistake. I inadvertently changed the `>=` to `=` and I need to change it back. I think OMake 0.10.5 is still broken on OCaml 5.0 anyway.

**Comment by jhwoodyatt** [comment]
And now the failure in CI is with OMake 0.10.5 (same as with 0.10.3).

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**